guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -4,7 +4,7 @@
 Before installing {Project} in an IPv6 network, ensure that you meet the following requirements:
 
 ifdef::satellite[]
-* If you plan to provision hosts from {Project} or {SmartProxies}, you must install {Project} and {SmartProxies} on {RHEL} version 7.9 or higher because these versions include the latest version of the `grub2` package.
+* If you plan to provision hosts from {Project} or {SmartProxies}, you must install {Project} and {SmartProxies} on {RHEL} version 8.0 or higher because these versions include the latest version of the `grub2` package.
```

Comment:
Technically it's RHEL 8.3 where grub2 has the right fixes.
```suggestion
* If you plan to provision hosts from {Project} or {SmartProxies}, you must install {Project} and {SmartProxies} on {RHEL} version 8.3 or higher because these versions include the latest version of the `grub2` package.
```

In b9437c01aa8bd93ebe56ab8321abbc87f8bb1cbf we dropped the note altogether for non-Satellite and I wonder if we should do the same thing. RHEL 8.3 is quite old by now and Satellite only supports the latest RHEL (minor) version.